tests/release/release.go

```diff
@@ -134,4 +142,85 @@ var _ = framework.ReleaseSuiteDescribe("test-demo", func() {
 			Expect(release.Status.ReleasePipelineRun).Should(Equal(fmt.Sprintf("%s/%s", pipelineRun.Namespace, pipelineRun.Name)))
 		})
 	})
+
+	var _ = Describe("Failure- Missing matching release link", func() {
+		BeforeAll(func() {
+			// Create the dev namespace
+			sourceNamespace, err := framework.CommonController.CreateTestNamespace(failure1SourceNamespace)
+			Expect(err).NotTo(HaveOccurred(), "Error when creating namespace '%s': %v", sourceNamespace.Name, err)
+
+			// Create the managed namespace
+			managedNamespace, err := framework.CommonController.CreateTestNamespace(failure1ManagedNamespace)
+			Expect(err).NotTo(HaveOccurred(), "Error when creating namespace '%s': %v", managedNamespace.Name, err)
+
+			// wait until both namespaces have been created
```

Comment:
I think this commented out code should be removed

Reply:
and remove empty lines